apps/nextjs-example/src/utils/standardWallet.ts

```diff
@@ -221,16 +395,33 @@ export class MyWallet implements AptosWallet {
     });
   };
 
+  /**
+   * REVISION - Implement this function using your Wallet. 
+   * 
+   * An event which will be triggered anytime an Account changes.
+   * 
+   * @returns when the logic is resolved.
+   */
   onAccountChange: AptosOnAccountChangeMethod = async (): Promise<void> => {
+    // THIS LOGIC SHOULD BE REPLACED. IT IS FOR EXAMPLE PURPOSES ONLY.    
     return Promise.resolve();
   };
 
+  /**
+   * REVISION - Implement this function using your Wallet. 
+   * 
+   * When users indicate a Network change should occur, update your Wallet accordingly.
+   * 
+   * @returns when the logic is resolved.
+   */
   onNetworkChange: AptosOnNetworkChangeMethod = async (): Promise<void> => {
+    // THIS LOGIC SHOULD BE REPLACED. IT IS FOR EXAMPLE PURPOSES ONLY.    
     return Promise.resolve();
   };
 }
 
-// a wallet call to register itself (this happens on the wallet side)
+// This section should NOT be revised (although it may make sense to move where it gets called).
```

Comment:
so this section is (only) for chrome extension wallets (the wallets we need to install from the chrome store). If the wallet is a SDK wallet (npm package) there is no need to fire a registerWallet event.

We use it here just so the demo dapp can be aware of this wallet example

Reply:
Is there a separate example of an SDK wallet template? (How does the npm package version register itself?)

(Maybe we can chat about this later in a bit more depth?)

Reply:
The SDK/npm package adds itself to the adapter code https://aptos.dev/en/build/sdks/wallet-adapter/wallets#is-your-wallet-a-sdk-npm-package-wallet